components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -207,6 +221,34 @@ export default function TeamSettingsPage() {
                         />
                     </ConfigurationSettingsField>
 
+                    {isCommitAnnotationEnabled && (
```

Comment:
Just curious, how we indicate what kind of settings should be on `General` and `Policies`? (I was trying to find this setting on Policies tab before look through the code)

Reply:
@mustard-mh I generally go with my gut here. This time around, I didn't consider this feature a policy, since it's not restricting anything per se, like our other policies we have on that page.

It could definitely be argued that "it's asserting that all of an organization's commits have our `Tool:` trailer", though, so maybe? What do **you** think?

Reply:
I don't have a strong opinion there, just curious 😋 Any place is good for me (Once the docs to the user is clear, and for sure you will do it)